Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM python:3.9.0-alpine
```

Comment:
This aims to reduce the size of images from ~1G to ~180M

Reply:
Great change.